src/pe_reports/report_gen/forms.py

```diff
@@ -0,0 +1,41 @@
+"""Create the stakeholder data input form."""
+
+# Third-Party Libraries
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField, TextAreaField
+
+
+class InfoFormExternal(FlaskForm):
+    """Create web form to take user input on report to be generated."""
+
+    report_date = StringField(
+        "What is the report date? (Final day of the report period, either the 15th or last day of the month)"
+        "*format YYYY-MM-DD"
+    )
+    output_directory = StringField(
+        "The directory where the final PDF reports should be saved. "
+    )
+
+    submit = SubmitField("Submit", render_kw={"onclick": "loading()"})
+
+
+class BulletinFormExternal(FlaskForm):
+    """Create web form to take user input on bulletin to be generated."""
+
+    cybersix_id = StringField("Cybersix Intel Item ID:")
+    user_input = TextAreaField(
+        "Please provide an explanation of what was found in the post/intel_item."
+    )
+    output_directory1 = StringField("Output Directory:")
+    file_name = StringField("File Name?")
+
+    submit1 = SubmitField("Submit", render_kw={"onclick": "loading()"})
+
+
+class CredsFormExternal(FlaskForm):
+    """Create web form to take user input on bulletin to be generated."""
+
+    org_id = StringField("Organization Cyhy ID:")
+    breach_name = StringField("Breach Name:")
+
+    submit2 = SubmitField("Submit", render_kw={"onclick": "loading()"})
```

Comment:
Is there a reason that you need/want to add 1 or 2 after these variable names?  Since they are in different classes, I'd think that this is sufficient and cleaner:
```suggestion
    output_directory = StringField("Output Directory:")
    file_name = StringField("File Name?")

    submit = SubmitField("Submit", render_kw={"onclick": "loading()"})


class CredsFormExternal(FlaskForm):
    """Create web form to take user input on bulletin to be generated."""

    org_id = StringField("Organization Cyhy ID:")
    breach_name = StringField("Breach Name:")

    submit = SubmitField("Submit", render_kw={"onclick": "loading()"})
```

Obviously if you accept this suggestion, don't forget to make the corresponding changes wherever those variable names are referenced.

Reply:
I didn't think it was necessary either, but when I had both forms on the same page in the UI we had an issue with them being named the same. We followed the solution in this stack overflow post: https://stackoverflow.com/questions/18290142/multiple-forms-in-a-single-page-using-flask-and-wtforms

Reply:
OK, though I think it's worth adding a comment explaining why you have to do that.

Reply:
@dav3r comments added in the following commit: [b8f6652](https://github.com/cisagov/pe-reports/pull/282/commits/b8f665281d5b2bfa175ef2007cdb32b497d37ee1)